Dockerfile

```diff
@@ -0,0 +1,81 @@
+# base image
+FROM ruby:2.3.0
+
+# set environment variables
+ENV \
+  APP_HOME=/legislated \
+  PG_VERISON=9.6 \
+  PHANTOM_JS_VERSION=phantomjs-2.1.1-linux-x86_64
+
+RUN \
```

Comment:
This has gotten pretty long have you thought about putting it in a script like `scripts/docker_setup.sh`